### PR TITLE
Update squad label to console

### PR DESCRIPTION
**Pull request by ycao56**
Signed-off-by: Yu Cao <ycao@redhat.com>

### CODE REVIEW

File: tests/cypress/tests/all_policies_summary_table.spec.js [hunk 1]
```diff
@@ -10,7 +10,7 @@
 import { describeT } from '../support/tagging'
 import { getConfigObject } from '../config'
 
-describeT('RHACM4K-2343 - [P1][Sev1][policy-grc] All policies page: Verify summary table', () => {
+describeT('RHACM4K-2343 - [P1][Sev1][console] All policies page: Verify summary table', () => {
```
Comment (KevinFCormier) [inline tests/cypress/tests/all_policies_summary_table.spec.js:13]:
```suggestion
describeT('RHACM4K-2343 - GRC UI: [P1][Sev1][console] All policies page: Verify summary table', () => {
```

File: tests/cypress/tests/ansible_automation.spec.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ import { describeT } from '../support/tagging'
 import { getConfigObject } from '../config'
 import { getDefaultSubstitutionRules } from '../support/views'
 
-describeT('@bvt RHACM4K-3471 - [P1][Sev1][policy-grc] All policies page: Verify automation modal', () => {
+describeT('@bvt RHACM4K-3471 - [P1][Sev1][console] All policies page: Verify automation modal', () => {
```
Comment (KevinFCormier) [inline tests/cypress/tests/ansible_automation.spec.js:14]:
```suggestion
describeT('@bvt RHACM4K-3471 - GRC UI: [P1][Sev1][console] All policies page: Verify automation modal', () => {
```

**Comment by ChunxiAlexLuo** [comment]
> @ycao56: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:
> 
> Test name	Commit	Details	Required	Rerun command
> ci/prow/e2e-tests	[9fcd815](https://github.com/open-cluster-management/grc-ui/commit/9fcd815e60f47efa2df27be44824b9bcbae8e22e)	[link](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/open-cluster-management_grc-ui/812/pull-ci-open-cluster-management-grc-ui-main-e2e-tests/1448358707065786368)	true	`/test e2e-tests`
> [Full PR test history](https://prow.ci.openshift.org/pr-history?org=open-cluster-management&repo=grc-ui&pr=812). [Your PR dashboard](https://prow.ci.openshift.org/pr?query=is:pr+state:open+author:ycao56).

It is an issue that https://github.com/open-cluster-management/grc-ui/pull/804 is trying to fix, current just retest here

**Comment by ycao56** [comment]
@KevinFCormier I am looking to merge this tonight (before code cut off) so that we don't have to back port to 2.4.

**Pull request closed without merging**